tests/rest_swagger_server_test.cc

```diff
@@ -13,7 +13,13 @@
 #include <pistache/router.h>
 #include <pistache/serializer/rapidjson.h>
 
+#if ___GNUG__ <= 7
+#include <experimental/filesystem>
+namespace filesystem = std::experimental::filesystem;
+#else
 #include <filesystem>
+namespace filesystem = std::filesystem;
+#endif
 #include <httplib.h>
 
 using namespace std;
```

Comment:
Same as above, but note that here you rename `std::filesystem` to `filesystem`, and then `using namespace std;` is used, causing namespace clashes. Please change `namespace filesystem = ...` to `namespace fs = ...`, and then update the test code (and possibly run tests locally to make sure you don't break anything :)

Reply:
I test clang 6.0/10.0 with gcc 7.5.0, it works fine.
Can you tell me your compilation environment?